docs/sources/helm-charts/mimir-distributed/release-notes/v4.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir Helm chart version 4.3 release notes"
+menuTitle: "V4.3 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 4.3"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 4.3 release notes
+
+Grafana Labs is excited to announce version 4.3 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.7 or Grafana Enterprise Metrics (GEM) v2.7. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.7](/docs/mimir/v2.4.x/release-notes/v2.7/) or [Grafana Enterprise Metrics v2.7](/docs/enterprise-metrics/latest/release-notes/v2-7/).
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
```

Comment:
Let's keep the note from 4.1

```suggestion
> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.

```